packages/turf-line-offset/index.js

```diff
@@ -103,12 +103,31 @@ function lineOffsetFeature(line, distance, units) {
  * @returns {Array<Array<number>>} offset points
  */
 function processSegment(point1, point2, offset) {
-    var L = Math.sqrt((point1[0] - point2[0]) * (point1[0] - point2[0]) + (point1[1] - point2[1]) * (point1[1] - point2[1]));
+    // arccos of dot product between point1 and point2 is the angle between points (in radians)
+    var angle = Math.acos(
+        Math.cos(point1[1]*Math.PI/180)*Math.cos(point1[0]*Math.PI/180) * Math.cos(point2[1]*Math.PI/180)*Math.cos(point2[0]*Math.PI/180)
+        + Math.cos(point1[1]*Math.PI/180)*Math.sin(point1[0]*Math.PI/180) * Math.cos(point2[1]*Math.PI/180)*Math.sin(point2[0]*Math.PI/180)
+        + Math.sin(point1[1]*Math.PI/180) * Math.sin(point2[1]*Math.PI/180)
+    );
```

Comment:
It might be good to factor out `Math.PI/180` into its own constant so it doesn't need to be recalculated all the time.

Reply:
Actually I think we're doing duplicate cos and sin computations here as well that we should probably factor out.